src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -636,10 +636,19 @@ const Prescriptions = () => {
         ) : (
           <>
             <CernerFacilityAlert />
-            {paginatedPrescriptionsList &&
-            paginatedPrescriptionsList.length === 0 ? (
+            {(!showFilterContent &&
+              paginatedPrescriptionsList &&
+              paginatedPrescriptionsList.length === 0) ||
+            (showFilterContent &&
+              filteredList &&
+              filteredList?.length === 0 &&
```

Comment:
is there any reason to do filteredList && filteredList?.length (not the question mark in the second condition)

Reply:
good point. I was just following the logic above it but yeah I didn't do it below. I'll update it. Thank you